pkgs/applications/science/biology/mrtrix/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
+, fetchFromGitHub
+, python2Packages
+, eigen
+, mesa
+, qt56
+, x11
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  name = "MRtrix3-${version}";
+  version = "0.3.16";
+
+  src = fetchFromGitHub {
+    owner = "MRtrix3";
+    repo = "mrtrix3";
+    rev = "${version}";
+    sha256 = "1h2r1i6qs09cpb476cqwk5fcq7q2g4cqrzadavvf3bz66dr81ir8";
+  };
+
+  nativeBuildInputs = [
+    qt56.qmake
+    qt56.qtscript
```

Comment:
`qmake` is a `nativeBuildInput` because it is a build tool, but `qtscript` is a runtime dependency that should go in `buildInputs`.